### ISSUE TITLE
firefox-with-plugins wrapper fails to make MOZ_PLUGIN_PATH exports

### ISSUE THREAD

**Issue by kalebo** [NixOS/nixpkgs#11199]
My last nixos-rebuild broke the flash and googletalk plugins for both firefox 41.0.2 and 42.0. The problem can be seen in the wrapper for firefox, where in previous working versions the wrapper script included lines like this:
```
export MOZ_PLUGIN_PATH=$MOZ_PLUGIN_PATH${MOZ_PLUGIN_PATH:+:}/nix/store/ ... -flashplayer-11.2.202.508/lib/mozilla/plugins
export MOZ_PLUGIN_PATH=$MOZ_PLUGIN_PATH${MOZ_PLUGIN_PATH:+:}/nix/store/ ... -google-talk-plugin-5.4.2.0/lib/mozilla/plugins

```
However, these export lines are no longer present in builds downloaded from the unstable channel and consequently firefox has no plugins. 

I have built firefox 42.0 from source by cherry-picking the relevant commits with the same results. The nix expressions I used can be seen here: https://github.com/NixOS/nixpkgs/compare/master...kalebo:moz_plugins . 

I am running NixOS unstable version 16.03pre69762.e916273 (Emu). Please let me know if I've missed something obvious.

**Comment by nbp** [comment]
cc @edolstra

**Comment by ttuegel** [comment]
I don't know when this problem was introduced, but it is now resolved in both `master` and `release-16.03`.
